target/linux/kirkwood/image/Makefile

```diff
@@ -102,11 +97,19 @@ define Device/linksys
   UBINIZE_OPTS := -E 5
   KERNEL_SIZE := 3072k
   IMAGES := sysupgrade.bin
+  DEVICE_COMPAT_VERSION := 2.0
+  DEVICE_COMPAT_MESSAGE := Config cannot be migrated from swconfig to DSA. \
+	Partition layout has changed compared with older versions (up to \
+	19.07) due to kernel size restrictions. Sysupgrade needs to be done \
+	in a later version than 19.07, which can be booted from a USB drive. \
+	Alternatively, perform a new installation via TFTP in the U-Boot. \
+	For instructions, see your device's page on https://openwrt.org/toh. \
+	Warning: Force upgrade (sysupgrade -F | --force) will likely brick \
+	this device!
```

Comment:
DSA comment would be irrelevant now.

And I don't see why you would add this here, where all Linksys device are and will be affected, even new ones.